recipes/sesh/recipe.yaml

```
@@ -0,0 +1,48 @@
+context:
+  name: sesh
+  version: "2.15.0"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/joshmedeski/${{ name }}/archive/v${{ version }}.tar.gz
```

```suggestion
  url: https://github.com/joshmedeski/sesh/archive/v${{ version }}.tar.gz
```
Usually we prefer not to use jinja in URLs aside from versions to make the URLs easier for human readability